Dfe.Academies.External.Web/Enums/AcademisationWorkflowEnums.cs

```diff
@@ -44,4 +44,12 @@ public enum KeyPersonRole
     Trustee = 4,
     [Description("Other")]
     Other = 5
+}
+
+public enum SelectOption
+{
+	[Description("Yes")]
```

Comment:
Mixed tabs and spaces have crept in here somehow

Reply:
because I copied this enum from A2c-sip I think. I do have a number of code clean-up fixes which I'm going to do separate branches / PR's for